recipe/meta.yaml

```diff
@@ -27,22 +27,22 @@ outputs:
       noarch: python
     requirements:
       host:
-        - python >=3.9
+        - python {{ python_min }}
```

Comment:
```suggestion
        - python >=3.10
```

Reply:
Yeah, reading through CFEP-25 I'm not sure I understand the logic of requiring the minimum version on the host? 🤔

Reply:
See discussion below.

Reply:
AFAICT the discussion is on updating the python_min pin, but the bot PR here and elsewhere removes the `>=` in favour of just requiring the minimum version -- I don't understand this (unless I've misunderstood the syntax).

Reply:
See my comment in the main PR thread: for another PR, this was set to `>=` so the bot is doing something odd.

Reply:
Oops... ^h^h ... the change that I referred to was in `run` and not in `host`: there it is also pinned to `python {{ python_min }}`.